README.md

```diff
@@ -160,3 +160,16 @@ Coding style is One True Brace style. In astyle:
 
     astyle --mode=java --style=1tbs -U -H -I -R 'browse-handler/*' 'browse-indexing/*' 'common/*' 'tests/org/*'
 
+
+6.  Migration from earlier releases
+-----------------------------------
+
+Earlier versions of the browse handler used different names for
```

Comment:
We could specifically say "versions included in VuFind® 9,x and earlier" -- since the goal is to get this PR merged in time for release 10.0.

Reply:
I've just added a note for this, thanks!